### PR TITLE
Display content for job items.

### PR DESCRIPTION
**Pull request by jaden**
I'd like a cleaner way to toggle the 'No comments yet'
message, but I don't know how (or if it's possible) to access
Item.isJob() from ItemView.

### CODE REVIEW

File: src/components/Item.vue [hunk 1]
```diff
@@ -17,6 +17,7 @@
         | <a :href="'#/item/' + item.id">{{item.descendants}} {{item.descendants | pluralize 'comment'}}</a>
       </span>
     </p>
+    <p class="jobtext" v-show="isJob" v-html="item.text"></p>
```
Comment (yyx990803) [inline src/components/Item.vue]: Let's use `v-if` here since this is not going to change at runtime

Comment (jaden) [inline src/components/Item.vue]: Ah, I wasn't quite clear on the distinction. Thanks for the clarification.

Comment (yyx990803) [inline src/components/Item.vue]: There's some more details here: http://vuejs.org/guide/conditional.html#v-if_vs-_v-show

**Comment by jaden** [comment]
I also just realized I had been testing the individual item listings, but the news listing shows the full content of a job listing.

**Pull request merged**